front/src/app/pages/home-page/home-page.component.html

```diff
@@ -1,2 +1,8 @@
-<p>home-page works!</p>
-<app-logout></app-logout>
+<body>
+  <div class="home-page-container">
+    <app-banner></app-banner>
+    <div class="welcome"><h3>Welcome, {{username}}!</h3></div>
```

Comment:
add an ngif condition to avoid empty username display

Reply:
No <3